node/index.ts

```diff
@@ -54,7 +55,7 @@ export default new Service<Clients, State, ParamsContext>({
   },
   events: {
     broadcasterNotification: [
-      settings, errors, throttle, locale, notify,
+      stockFilterEvents, settings, errors, throttle, locale, notify,
```

Comment:
Maybe change the name to: `filterStockEvents` (verb, object). I think it makes more clear what is the purpose of the middleware.

Reply:
Fixed file and middleware name